code/game/objects/items.dm

```diff
@@ -117,6 +117,10 @@ GLOBAL_DATUM_INIT(fire_overlay, /image, image("icon" = 'icons/goonstation/effect
 	var/in_inventory = FALSE //is this item equipped into an inventory slot or hand of a mob?
 	var/tip_timer = 0
 
+	/// item hover FX
+	var/in_storage = FALSE
+	var/outline_filter
```

Comment:
Consider documenting this variable, as it's not apparent what it does at a glance.